geolocator_platform_interface/lib/src/geolocator_platform_interface.dart

```diff
@@ -159,6 +159,15 @@ abstract class GeolocatorPlatform extends PlatformInterface {
     throw UnimplementedError('getPositionStream() has not been implemented.');
   }
 
+  /// Returns a [Future] containing a [LocationAccuracyStatus]
+  /// When on iOS the user has given permission for approximate location,
+  /// [LocationAccuracyStatus.reduced] will be returned, if the user gave
+  /// permission for precise/full accuracy location, [LocationAccuracyStatus.precise]
+  /// will be returned
```

Comment:
nit: Missing periods and needs an empty line:

```suggestion
  /// Returns a [Future] containing a [LocationAccuracyStatus].
  ///
  /// When on iOS the user has given permission for approximate location,
  /// [LocationAccuracyStatus.reduced] will be returned, if the user gave
  /// permission for precise/full accuracy location, [LocationAccuracyStatus.precise]
  /// will be returned.
```